clients/cli/Cargo.toml

```diff
@@ -1,18 +1,10 @@
 [package]
-name = "lockbook-cli"
-version = "0.5.10"
+name = "cli2"
+version = "0.1.0"
 edition = "2021"
 
-[[bin]]
-name = "lockbook"
-path = "src/main.rs"
```

Comment:
probably do want the og values here

Reply:
oops

Reply:
https://github.com/lockbook/lockbook/pull/1561/commits/f153fc09e0e028af588cfd7b232af04f02a757ff